# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/cert_storage/Cargo.toml
# Commit: 16350e08d34f
# Full Hash: 16350e08d34ff06c7f46745c6b160c7a7bda9140
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2019-12-06 04:43:57
# Regressor Bug: 1586855
# File Overlap Count: 6
# Description:
#   bug 1586855 - incorporate CRLite filters into cert_storage r=jcj,kjacobs
#   
#   This patch implements CRLite lookups for TLS server certificate revocation
#   information in telemetry-only mode. It adds a new preference
#   "security.pki.crlite_mode" to control the behavior of this feature. Setting
# ==============================================================================

diff -r 64fd40663930 -r 16350e08d34f security/manager/ssl/cert_storage/Cargo.toml
--- a/security/manager/ssl/cert_storage/Cargo.toml	Thu Dec 05 22:23:54 2019 +0000
+++ b/security/manager/ssl/cert_storage/Cargo.toml	Thu Dec 05 22:41:53 2019 +0000
@@ -8,11 +8,13 @@
 byteorder = "1.2.7"
 crossbeam-utils = "0.6.3"
 log = "0.4"
+memmap = "0.7"
 moz_task = { path = "../../../../xpcom/rust/moz_task" }
 nserror = { path = "../../../../xpcom/rust/nserror" }
 nsstring = { path = "../../../../xpcom/rust/nsstring" }
+rental = "0.5.5"
 rkv = { git = "https://github.com/mozilla/rkv", rev="6a866fdad2ca880df9b87fcbc9921abac1e91914", default-features = false }
-rust_cascade = "0.3.4"
+rust_cascade = "0.5.0"
 sha2 = "^0.8"
 storage_variant = { path = "../../../../storage/variant" }
 tempfile = "3"